src/Toolkit/Toolkit/UI/Controls/MeasureToolbar/MeasureToolbar.cs

```diff
@@ -587,15 +590,15 @@ private static void OnMapViewPropertyChanged(DependencyObject d, DependencyPrope
 
             newMapView.GeoViewTapped += toolbar.OnMapViewTapped;
             toolbar._originalSketchEditor = newMapView.SketchEditor;
-            toolbar.DisplayResult(newMapView.SketchEditor.Geometry);
+            toolbar.DisplayResult(newMapView.SketchEditor?.Geometry);
         }
 
         /// <summary>
         /// Gets or sets the sketch editor used for measuring distances.
         /// </summary>
-        public SketchEditor LineSketchEditor
+        public SketchEditor? LineSketchEditor
```

Comment:
Can LineSketchEditor and AreaSketchEditor properties be non-null? They are initialized in the constructor and their PropertyChangedCallback prevent assigning null values.

Reply:
Yes dependency properties can always be null (just set it to null in a try/catch)